contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -398,6 +406,10 @@ ProcessTDSRequest(TDSRequest request)
 				Assert(0);
 				break;
 		}
+
+		/* Disabling statement timeout after processing */
```

Comment:
Here the comment should end with full stop.
if the sentence starts with a small case letter then there is no need of a full stop